### PR TITLE
[Add my project ] Piano

### PR DESCRIPTION
**Pull request by jayk-gupta**
GitHub username: jayk-gupta

Please add my piano project made using html, css and javascript.

I hope I made the right template in the yml file, if there are any mistakes kindly let me know.

### CODE REVIEW

File: _data/projects/jayk-gupta-piano.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: Piano
+desc: Made a piano using Html, CSS and vanilla JavaScript,designed using vanilla CSS.You can play it both with mouse nad keyboard.
+site: https://github.com/jayk-gupta/Piano
+tags:
+  - html
+  - css
+  - javascript
+upforgrabs:
+  name: up for grabs
+  link: https://github.com/jayk-gupta/project/labels/up%20for%20grabs
```
Comment (ritwik12) [inline _data/projects/jayk-gupta-piano.yml]:
```suggestion
  link: https://github.com/jayk-gupta/Piano/labels/good%20first%20issue
```
You need to create a label in your repo "up for grabs" or use some other existing label such as "good first issue"

**Review by ritwik12 — DISMISSED** [review]

File: _data/projects/jayk-gupta-piano.yml (hunk 1 above)
Comment (jayk-gupta) [reply _data/projects/jayk-gupta-piano.yml]: resolved

**Comment by jayk-gupta** [comment]
made the changes

**Comment by shiftkey** [comment]
@jayk-gupta I've added the `needs tasks` label here as there are no issues listed on the repository that are using this label. Are you able to add any tasks that would be ideal for new contributors to the project?

File: _data/projects/jayk-gupta-piano.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+name: Piano
+desc: Made a piano using Html, CSS and vanilla JavaScript,designed using vanilla CSS.You can play it both with mouse nad keyboard.
+site: https://github.com/jayk-gupta/Piano
+tags:
+  - html
+  - css
+  - javascript
+upforgrabs:
+  name: up for grabs
```
Comment (shiftkey) [inline _data/projects/jayk-gupta-piano.yml]: :art: this is what the `name` here should be

```suggestion
  name: good first issue
```

**Review by shiftkey — DISMISSED** [review]

**Comment by jayk-gupta** [comment]
> @jayk-gupta I've added the `needs tasks` label here as there are no issues listed on the repository that are using this label. Are you able to add any tasks that would be ideal for new contributors to the project?

YES!

**Pull request closed without merging**